Update README.md (#4)
bharel/mkdocs-render-swagger-plugin: README.md

# mkdocs-render-swagger-plugin

> This is the [Mkdocs](https://mkdocs.org) plugin for rendering swagger &amp; openapi schemas using Swagger UI. It is written in Python.

Reviewer: ```suggestion
This is the [Mkdocs](https://mkdocs.org) plugin for rendering swagger &amp; openapi schemas using [Swagger UI](https://swagger.io/tools/swagger-ui/). It is written in Python.
```

    
Enter `!!swagger <filename>!!` at the appropriate location inside a markdown file.

> ## Running Locally
> 
> You can install this package locally using `pip` and the `--editable` flag used for making developing Python packages.

Reviewer: ```suggestion
## Contributing & Developing Locally

After downloading and extracting the `.tar.gz`, install this package locally using `pip` and the `--editable` flag:
```

```bash
pip install --editable .
```

> You'll then have the `render-swagger` package available to use in Mkdocs and `pip` will point the dependency to this folder.

Reviewer: ```suggestion
You'll then have the `render-swagger` package available to use in Mkdocs and `pip` will point the dependency to this folder. You are then able to run the docs using `mkdocs serve`. Make sure you restart the process between code changes as the plugin is loaded on startup.
```

## MkDocs plugins and Swagger api

> The Render Swagger MkDocs plugin comes with a set of extensions and plugins that mkdocs and Swagger UI supports.

Reviewer: ```suggestion
The Render Swagger MkDocs plugin uses a set of extensions and plugin APIs that MkDocs and Swagger UI supports.
```

> You can find a list of all the MkDocks plugins and Swagger UI that are included in the Render Swagger plugin on the official website of [MkDocks](https://www.mkdocs.org/user-guide/plugins/) and [SwaggerUI](https://github.com/swagger-api/swagger-ui/blob/master/docs/customization/plugin-api.md)

Reviewer: ```suggestion
You can find more info about MkDocks plugins and Swagger UI  on the official website of [MkDocks](https://www.mkdocs.org/user-guide/plugins/) and [SwaggerUI](https://github.com/swagger-api/swagger-ui/blob/master/docs/customization/plugin-api.md).
The input OpenAPI files processed by the plugin should conform to the [OpenAPI specification](https://swagger.io/specification/). It is generated by a few projects such as [pydantic](https://pydantic-docs.helpmanual.io/), [FastAPI](https://fastapi.tiangolo.com/) and others.
```
